docs/technical_documents/open_space_decider.md

```diff
@@ -0,0 +1,180 @@
+# OPEN SPACE DECIDER
+
+_**Tip**: to read the equations in the document, you are recommended to use Chrome with [a plugin](https://chrome.google.com/webstore/detail/tex-all-the-things/cbimabofgmfdkicghcadidpemeenbffn) or copy the latex equation to [an online editor](http://www.hostmath.com/)_
```

Comment:
Get rid of it, if no latex equations are written @spicyog @JackFu123 @liouyx @xiaoluauto